data/templates/src/pe/dll_gdiplus/template.c

```diff
@@ -50,7 +50,7 @@ void ExecutePayload(void) {
 	si.cb = sizeof(si);
 
 	// Create a suspended process, write shellcode into stack, make stack RWX, resume it
-	if(CreateProcess( 0, "rundll32.exe", 0, 0, 0, CREATE_SUSPENDED|IDLE_PRIORITY_CLASS, 0, 0, &si, &pi)) {
+	if(CreateProcess( 0, "rundll32.exe", 0, 0, 0, CREATE_SUSPENDED|IDLE_PRIORITY_CLASS|CREATE_BREAKAWAY_FROM_JOB, 0, 0, &si, &pi)) {
```

Comment:
This change makes a lot of sense. Is it compiled into a new binary though, because if so I'm not sure I see it.

Reply:
Haven't compiled this one into a new binary, will do that now though 👍

Reply:
Interestingly this directory, `dll_gdiplus` has some very unusual compilation where half of it is compiled on Linux and half on Windows. The other compilation for `dll_mixed_mode` is currently failing due to `mscoree.lib` not being found, and it doesn't seem to exist on the system. Might want to consider if we should update the build processes for these DLLs to modernize them a bit.

Reply:
Did you just `fork()` windows? :wink:

Reply:
Removed this file from PR as its not strictly related to this PR and is likely better placed in a dedicated PR.